docs/rsdl/rapid-pro-rsdl-abnf.md

```diff
@@ -6,84 +6,91 @@ title: RAPID SDL ABNF
 # RAPID Pro syntax
 
 > DRAFT
-Initial Draft. July 2020
-
+> October 2020
 
 ## Overview
 
 This grammar uses ABNF as defined by [RFC5234](https://tools.ietf.org/html/rfc5234).
 
-Note: whitespace is not (yet) reflected in the grammar.
+Note: to increase readability of the grammer
+
+- whitespace is not reflected
+- double-quoted literals are case-_sensitive_
 
 ## Syntax rules
 
 - [Model](#model)
-- [Type definition](#type-definition)
-- [Enum definition](#enum-definition)
-- [Service definition](#service-definition)
-- [Core syntax elements](#core-syntax-elements)
+- [Structured Type](#structured-type)
+- [Enumeration Type](#enumeration-type)
+- [Service](#service)
+- [Core Syntax Elements](#core-syntax-elements)
 
 ### Model
 
 ```ABNF
-model               = 1*modelElement
+model               = [ namespace ] *modelElement
 
-modelElement        = typeDefinition
-                    / serviceDefinition
-                    / enumDefinition
+namespace           = "namespace" qualifiedName
+
+modelElement        = structuredType / enumType / service
 ```
 
-### Type definition
+### Structured Type
```

Comment:
+1 on "structured" 
The suffix "definition" is something that I feel belongs to the title and the name of the syntax rule. It is allows to easily distinguish between defining a type vs. using the type.